### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by cleptric**

### CODE REVIEW

**Comment by cleptric** [comment]
Well, that's interesting. I have no idea why all of the sudden the whole file changed.

**Comment by ndm2** [comment]
All line endings changed to `CRLF`.

**Comment by dereuromark** [comment]
Windows people :)

**Comment by dereuromark** [comment]
Or maybe a git setting (default) here on local system?

**Comment by cleptric** [comment]
I actually did this change via the web UI of Github.

File: .gitignore [hunk 1]
```diff
@@ -1,45 +1,53 @@
...
+# OS generated files #
+######################
+.DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+# Icon must end with two \r
+Icon
+
```
Comment (garas) [inline .gitignore]: You've also lost `\r` here, comment warns to be careful here

Comment (markstory) [reply .gitignore]: The mixed line endings are probably why github UI switched the line ending on you.

**Comment by cleptric** [comment]
Looks better now :)

**Pull request merged**